pkg/daemon/admission/server.go

```diff
@@ -54,24 +53,13 @@ func New(context *clusterd.Context, providerName string, validator admitFunc) *A
 // StartServer will start the server
 func (a *AdmissionController) StartServer() {
 	logger.Infof("starting the webhook for backend %q", a.providerName)
-	certPath := filepath.Join(tlsDir, tlsCertFile)
-	keyPath := filepath.Join(tlsDir, tlsKeyFile)
-	keyPair, err := NewTlsKeypairReloader(certPath, keyPath)
```

Comment:
Are tlsutil.go and controller.go used anymore? Can we remove them? If so, then perhaps this package is so simple we can delete the whole package and move the remaining code under the ceph provider. For example, similar to what was done for the nfs operator in pkg/operator/nfs/webhook.go.

Reply:
This is a draft PR. There are still  some issues i am working on. I'll let you know once the updates are done

Reply:
I've removed the tlsutil.go and controller.go

Reply:
@travisn the code from the package (server.go) is meant to a generic server code that can work for all providers. I think it would make sense to move it under pkg/operator/admission/server.go. May add another file later in this package  for the unit tests. WDYT ?

Reply:
The generic code is so short now that I wonder if it makes sense to have any common code. The controller runtime packages already have the common code.

Reply:
Ok, i've moved the server code under pkg/operator/ceph/server.go